### DISCUSSION TITLE
2.8.6 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4686] [讨论]
· 更新 8 月赞助者列表，非常感谢大家的支持！
· 修复了以下 2 个 Bug
　　在搜索时删除 Mod 后报错（#4663，@milan-a11y）
　　安装 CurseForge 整合包时无法使用镜像源（#4665，@lihansen136）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.6'>2.8.6</a>.</em>

**Comment by jian2486** [comment]
不知道为什么加了个材质包和光影进入服务器就看不见除了正版玩家的皮肤了（BLS和红石显示)

**Reply by jian2486** [reply]
SkinsRestorer的那个插件

**Reply by tangge233** [reply]
Try CustomSkinLoader mod

**Reply by allMagicNB** [reply]
没看懂，你是服主还是玩家？

**Reply by MoYuan-CN** [reply]
请单独开启一个 Discussion 帖子讨论，更新日志不是你提问的地方，除非在更新之后才有此问题，感谢合作

**Comment by jian2486** [comment]
还是那样，修不了

**Reply by tangge233** [reply]
那你尝试用hmcl启动看看会不会出现类似的问题，如果有那就是游戏的问题。
